examples/configmapgen_controller.rs

```diff
@@ -112,16 +116,45 @@ async fn main() -> Result<()> {
     let cmgs = Api::<ConfigMapGenerator>::all(client.clone());
     let cms = Api::<ConfigMap>::all(client.clone());
 
-    Controller::new(cmgs, ListParams::default())
-        .owns(cms, ListParams::default())
-        .run(reconcile, error_policy, Context::new(Data { client }))
-        .for_each(|res| async move {
-            match res {
-                Ok(o) => info!("reconciled {:?}", o),
-                Err(e) => warn!("reconcile failed: {}", Report::from(e)),
-            }
-        })
-        .await;
+    log::info!("starting configmapgen-controller");
+    log::info!("press <enter> to force a reconciliation of all objects");
+    log::info!("press <ctrl>+c to shut down gracefully");
```

Comment:
i am amazed at how well you are managing to extend the stream paradigm here to control behaviour :+1: